Security/SessionDownloadStrategy.php

```diff
@@ -39,14 +42,31 @@ class SessionDownloadStrategy implements DownloadStrategyInterface
     protected $sessionKey = 'sonata/media/session/times';
 
     /**
-     * @param TranslatorInterface $translator
-     * @param ContainerInterface  $container
-     * @param int                 $times
+     * @var Session
      */
-    public function __construct(TranslatorInterface $translator, ContainerInterface $container, $times)
+    private $session;
+
+    /**
+     * @param TranslatorInterface        $translator
+     * @param ContainerInterface|Session $container
+     * @param int                        $times
+     */
+    public function __construct(TranslatorInterface $translator, $container, $times)
```

Comment:
You should try to make future diff of the non-BC PR minimal. Please rename `$container` here (and in the phpdoc too). As seen before, changing the variable name is not a BC-break